python/examples/integrations/writing_profiles.ipynb

```diff
@@ -0,0 +1,695 @@
+       "col_3          4            0               4                 0   \n",
+       "col_1          4            0               0                 4   \n",
+       "\n",
+       "        types/boolean  types/string  types/object  cardinality/est  \\\n",
+       "column                                                               \n",
+       "col_2               0             4             0              4.0   \n",
+       "col_3               0             0             0              4.0   \n",
+       "col_1               0             0             0              4.0   \n",
+       "\n",
+       "        cardinality/upper_1  cardinality/lower_1  ... distribution/n  \\\n",
+       "column                                            ...                  \n",
+       "col_2                4.0002                  4.0  ...            NaN   \n",
+       "col_3                4.0002                  4.0  ...            4.0   \n",
+       "col_1                4.0002                  4.0  ...            4.0   \n",
+       "\n",
+       "       distribution/max  distribution/min  distribution/q_10  \\\n",
+       "column                                                         \n",
+       "col_2               NaN               NaN                NaN   \n",
+       "col_3               5.0               2.0                2.0   \n",
+       "col_1               2.2               0.1                0.1   \n",
+       "\n",
+       "        distribution/q_25  distribution/median  distribution/q_75  \\\n",
+       "column                                                              \n",
+       "col_2                 NaN                  NaN                NaN   \n",
+       "col_3                 3.0                  4.0                5.0   \n",
+       "col_1                 1.0                  1.2                2.2   \n",
+       "\n",
+       "        distribution/q_90  ints/max  ints/min  \n",
+       "column                                         \n",
+       "col_2                 NaN       NaN       NaN  \n",
+       "col_3                 5.0       5.0       2.0  \n",
+       "col_1                 2.2       NaN       NaN  \n",
+       "\n",
+       "[3 rows x 24 columns]"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "profile_view = profile.view()\n",
+    "profile_view.to_pandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Cool! So now that we have a proper profile created, let's see how we can persist is as a file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Local writer\n",
+    "\n",
+    "The first and most straight-forward way of persisting a `whylogs` profile as a file is to write it directly to your disk. Our API makes it possible with the following commands. You could either write it from the `ProfileResultSet`"
```

Comment:
Maybe we could add the why.write(profile,"dest") way of writing, and then explain that this converts the profile into a profile view before writing it to disk.